core-feature-pack/common/src/main/resources/content/bin/standalone.bat

```diff
@@ -281,10 +281,8 @@ if not "%PRESERVE_JAVA_OPT%" == "true" (
             )
             "!JAVA!" !TMP_PARAM! -version > nul 2>&1
             if not errorlevel == 1 (
-               set "JAVA_OPTS=%JAVA_OPTS% !TMP_PARAM!"
+               set "JAVA_OPTS=!JAVA_OPTS! !TMP_PARAM!"
             )
-            rem Remove the gc.log file from the -version check
-            del /F /Q "%JBOSS_LOG_DIR%\gc.log" > nul 2>&1
```

Comment:
Oh interesting. Do you recall which test was failing?

Reply:
Sorry! my bad...I can't remember right now. If you say I can change back again to check the test.

Reply:
Sorry! my bad...I can't remember right now. If you say I can change back again to check the test.

Reply:
It's no problem. I can test it locally. I'm sure it was the standalone script test now that I think about it :) I'll test it locally as maybe the test is wrong.

Reply:
Thanks! my windows VM crashed somehow that's why I could not able to test it. Let me know if I need to change anything.

Reply:
Okay I think we should add it back. I just ran the script tests on my Windows VM with the delete added back and it worked fine. IIRC we added this delete because QE found it leaving that extra GC log behind.

Reply:
Thanks! Done :)